wagtail_streamfield_migration_toolkit/test/settings.py

```diff
@@ -111,6 +111,16 @@
     "default": dj_database_url.config(default="sqlite:///test_wagtail_streamfield_migration_toolkit.db"),
 }
 
+# DATABASES = {
```

Comment:
Why is this added? You set this above using `dj_database_url`

Reply:
oops that was not supposed to be committed